ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -300,20 +300,37 @@
       (= (:gtfs/route-hash-id-type route-hash-id-type) "short-long-headsign")
       (= (:gtfs/route-hash-id-type route-hash-id-type) "long-headsign")))
 
+(def selected-change-keys #{:removed-trips :trip-stop-sequence-changes-lower
+                            :trip-stop-sequence-changes-upper :route-hash-id
+                            :trip-stop-time-changes-lower :trip-stop-time-changes-upper :change-type :added-trips})
+
+(defn- list-route-changes-with-same-route-hash-id [all-changes single-change]
+  ;; Filter nil values
+  (keep
+    (fn [c]
+      (let [x (select-keys c selected-change-keys)
```

Comment:
Is `select-keys c selected-change-keys` redundant, because `list-route-changes-with-same-route-hash-id` gets a version of `all-changes` which already has gone through this same `select-keys`?

Reply:
Good catch, I'll check